cg_hermes/config/taxprofiler.py

```diff
@@ -0,0 +1,66 @@
+"""Tags that are defined in taxprofiler deliverables mapped to tags used in CG"""
+
+from cg_hermes.config.nextflow import NEXTFLOW_COMMON_TAGS
+
+TAXPROFILER_COMMON_TAGS = {
+    frozenset({"kraken2_combined_report", "kraken2"}): {
+        "is_mandatory": True,
+        "bundle_id": True,
+        "tags": ["kraken2", "kraken2report"],
```

Comment:
These tags are merely suggestions; feel free to use more convenient names for taxprofiler

```suggestion
        "tags": ["kraken2", "combined-report"],
```